# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-vcd/makefile-includedir.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vcd-0.9/Makefile.orig	2008-03-07 21:30:09.000000000 +0100
+++ vcd-0.9/Makefile	2008-03-07 21:39:08.000000000 +0100
@@ -47,7 +47,7 @@
 
 ### Includes and Defines (add further entries here):
 
-INCLUDES = -I$(VDRINC) -I$(DVBDIR)
+INCLUDES = -I$(VDRDIR)/include -I$(DVBDIR)
 
 DEFINES += -DPLUGIN_NAME_I18N='"$(PLUGIN)"' -D_GNU_SOURCE
 
